### PR TITLE
[SMALLFIX] Fix log message in FileSystemContext

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java [hunk 1]
```diff
@@ -250,7 +250,7 @@ private FileSystemContext(AlluxioConfiguration conf, @Nullable BlockWorker block
     mWorkerRefreshPolicy =
         new TimeoutRefresh(conf.getMs(PropertyKey.USER_WORKER_LIST_REFRESH_INTERVAL));
     LOG.debug("Created context with id: {}, with local block worker: {}",
-        mId, mBlockWorker == null);
+        mId, mBlockWorker != null);
```
Comment (ZhuTopher) [inline core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java:253]: Is this Java syntactic sugar for `mBlockWorker != null ? mBlockWorker : null` or are we intentionally logging a boolean expression result?

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for fixing the log message

File: core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java (hunk 1 above)
Comment (jiacheliu3) [reply core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java:253]: we are intentionally logging a boolean here as DefaultBlockWorker does not have a `toString()`.

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**